ui/components/MesheryApplications/ApplicationsCard.js

```diff
@@ -83,7 +83,10 @@ function MesheryApplicationCard({
                   e.stopPropagation(); handleAppDownload(id ,source_type, name)
                 }}
               >
-                <img src={`/static/img/${(source_type).replaceAll(" ", "_").toLowerCase()}.svg`} width="45px" height="45px" />
+                {source_type== undefined || source_type == " " ?
+                  <img src={`/static/img/default_apps-green.svg`} width="45px" height="45px" />
+                  : <img src={`/static/img/${(source_type).replaceAll(" ", "_").toLowerCase()}.svg`} width="45px" height="45px" />
```

Comment:
Static assignment of both height and width is not advised. Minimally, please only have one specified.

It'd be great to see flex used here and across the rest of Meshery UI's components.

Reply:
Unaddressed feedback remains.

Reply:
// @legendarykamal

Reply:
on it .